spec/system/user_preferences_navigation_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+describe 'Redesigned user page navigation menu', type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  let(:everyone_group) { Group[:everyone] }
+
+  describe "when visiting the user's preferences page with redesigned user page nav enabled" do
+    it 'should allow the user to scroll the horizontal navigation menu when window width is narrow' do
+      everyone_group.add(user)
+      SiteSetting.enable_new_user_profile_nav_groups = everyone_group.name
+
+      resize_window(width: 400) do
+        sign_in(user)
+
+        user_preferences_page = PageObjects::Pages::UserPreferences.new
```

Comment:
A thing that Nat did which I quite liked was just putting these pages into a `let` and reusing them, e.g. `let(:user_preferences_page) { PageObjects::Pages::UserPreferences.new }`